components/map/basic/src/leaflet/marker-manager.js

```diff
@@ -173,16 +178,19 @@ class MarkerManager {
   // Coupled FC, should be removed in the future
   addClassModifier(iconClassName, options) {
     const classModifiers = {
-      '--visited': this.isVisited,
+      '--contacted': this.isContacted,
       '--fav': this.isFavorite,
-      '--contacted': this.isContacted
+      '--visited': this.isVisited
     }
 
     const checkModifier = className => {
       return classModifiers[className](options)
     }
 
-    const modifier = Object.keys(classModifiers).find(checkModifier)
+    let modifier = Object.keys(classModifiers).find(checkModifier)
+    modifier = `${modifier} ${
+      this.isHighlighted(options) ? HIGHLIGHTED_CLASS : ''
+    }`
```

Comment:
I'd use another const instead of mutating a var:
```js
const matchingModifier = Object.keys(classModifiers).find(checkModifier)
const modifier = `${matchingModifier} ${
  this.isHighlighted(options) ? HIGHLIGHTED_CLASS : ''
}`
```